test/e2e/k8s_util.go

```diff
@@ -81,25 +82,24 @@ func (k *KubernetesUtils) GetPods(ns string, key string, val string) ([]v1.Pod,
 
 // Probe execs into a Pod and checks its connectivity to another Pod.  Of course it assumes
 // that the target Pod is serving on the input port, and also that ncat is installed.
-func (k *KubernetesUtils) Probe(ns1, pod1, ns2, pod2 string, port int32) (bool, error) {
-	fromPods, err := k.GetPods(ns1, "pod", pod1)
+func (k *KubernetesUtils) Probe(ns1, pod1, ns2, pod2 string, port int32, usePodName bool) (bool, error) {
```

Comment:
you forgot to remove `usePodName`? is there no warning from the code linters?

Reply:
my bad. Yeah the linters would not throw an error if a function param is not used